### PR TITLE
Upgrade version of Patternfly and Bump Version to 2.1.0

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
@spadgett

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 bower_components/*
-node_modules/*
\ No newline at end of file
+node_modules/*
+package-lock.json
```
Comment (spadgett) [inline .gitignore]: This file should actually be committed to my understanding

**Review by spadgett — COMMENTED** [review]
LGTM, but I think we should either commit package-lock.json or disable it with `package-lock=false` in .npmrc

**Comment by spadgett** [comment]
I'm OK committing package-lock.json unless you have a strong opinion @jeff-phillips-18

**Comment by jeff-phillips-18** [comment]
@spadgett  Yeah, I wasn't sure. I've added it.

**Review by spadgett — APPROVED** [review]

**Pull request merged**